src/connection/utils.ts

```diff
@@ -1,15 +1,21 @@
-export const isInjected = Boolean(window.ethereum)
+import { isMobile } from 'utils/userAgent'
```

Comment:
Can you minimize risk by mapping the logic 1:1, instead of changing the abstraction (ie introducing `isMobile` in this file, and changing the mobile logic in `src/connection/index.ts`)